### ISSUE TITLE
Cannot get `pt` binary

### ISSUE THREAD

**Issue by delphinus35** [Songmu/ghg#1]
This command below causes `unknown type flag` error.

```
$ ghg get -u monochromegane/the_platinum_searcher
fetch the GitHub release for monochromegane/the_platinum_searcher
install monochromegane/the_platinum_searcher version: v2.1.2
download https://github.com/monochromegane/the_platinum_searcher/releases/download/v2.1.2/pt_linux_amd64.tar.gz
[======================================] 1.61 MB/1.61 MB
extract pt_linux_amd64.tar.gz
failed to extract: pt_linux_amd64/pt: unknown type flag:
```

**Comment by Songmu** [comment]
Thank you for the report!

**Comment by Songmu** [comment]
I detect the problem and fix it in following pull request.

https://github.com/mholt/archiver/pull/6

After merging this pull request, I re-build the binaries. Please wait a while.

**Comment by Songmu** [comment]
Fixed and released as v0.0.2. Please try it!

**Comment by delphinus35** [comment]
Thx! I could get `pt` binary successfully.
